sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/RemoteModelFetcher.cs

```diff
@@ -22,22 +22,23 @@ internal class RemoteModelFetcher : IModelFetcher
     {
         private readonly HttpPipeline _pipeline;
         private readonly ClientDiagnostics _clientDiagnostics;
-        private readonly bool _tryExpanded;
+        private readonly ModelsRepositoryClientOptions _clientOptions;
```

Comment:
We don't need the client options here anymore.

Reply:
Removed.